Taskfile.yml

```diff
@@ -300,12 +297,25 @@ tasks:
     desc: Build & serve the playground.
     dir: web/playground
     cmds:
-      # Must set `install-links=false` in the playground's `npm install` to
-      # install prql-js as the regular dependency, instead of creating a
-      # symlink. Refer to https://github.com/PRQL/prql/pull/1296.
-      - npm install --install-links=false
+      - task: check-npm-cache
       - npm start
-#
+
+  check-npm-cache:
```

Comment:
Overall this is great, very nice addition.

Can we rename this to something like `install-playground`? It's doing more than just checking the cache.

Then let's merge!